server/lib/chat/chat-models.ts

```diff
@@ -304,12 +305,12 @@ export async function removeUserFromChannel(
       return {}
     }
 
-    const highTrafficChannelResult = await client.query(sql`
+    const officialChannelResult = await client.query(sql`
       SELECT id FROM channels
-      WHERE id = ${channelId} AND high_traffic = true;
+      WHERE id = ${channelId} AND official = true;
     `)
-    if (highTrafficChannelResult.rowCount > 0) {
-      // Don't transfer ownership in "high traffic" channels
+    if (officialChannelResult.rowCount > 0) {
```

Comment:
`rowCount` is actually not the right thing to use here (I learned this from doing stuff on the friends list). `rowCount` is intended to return the number of rows modified by a command (e.g. how many rows were `DELETE`d, `UPDATE`d, etc.). Use `officialChannelResults.rows.length` instead

Sort of relatedly, there's a bunch of queries in here that could return enough info to deal with all of these conditions, instead of having to run back to the db a ton of times, would probably be good to clean that up